crates/libcontainer/src/container/container_start.rs

```diff
@@ -88,10 +73,25 @@ impl Container {
         // Run post start hooks. It runs after the container process is started.
         // It is called in the runtime namespace.
         if let Some(hooks) = config.hooks.as_ref() {
+            let original_dir = env::current_dir().map_err(|err| {
+                tracing::error!("failed to get current directory: {}", err);
+                LibcontainerError::Other(format!("failed to get current directory: {}", err))
+            })?;
+
+            unistd::chdir(self.root.as_os_str()).map_err(|err| {
```

Comment:
Instead of chaging directory here and then reverting it, could we set `current_dir` in the new process instead?

Here: https://github.com/containers/youki/blob/c6fffb2c6a59b5fc2efc85189d485d322c5454e8/crates/libcontainer/src/hooks.rs#L39

```diff
-let mut hook_command = process::Command::new(hook.path());
+let mut hook_command = process::Command::new(hook.path())
+    .current_dir(root_dir);
```